tests/test_client.py

```diff
@@ -6,6 +6,9 @@
 import os
 import tempfile
 from unittest.mock import Mock
+import pandas as pd  # type: ignore
+import numpy as np  # type: ignore
+import io  # type: ignore
```

Comment:
io is standard library so should be above pandas and np
```suggestion
import io  # type: ignore

import pandas as pd  # type: ignore
import numpy as np  # type: ignore
```